app/views/locations/ips/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "layouts/form_errors", resource: @ip %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= link_to "Back to list", ips_path, class: "govuk-back-link" %>
+
+    <h2 class="govuk-heading-l">Add an IP address to <%= @location.address %></h2>
+
+    <%= render "ips/activation_notice" %>
+
+    <%= form_with model: @ip do |form| %>
+      <div class="govuk-form-group <%= field_error(@ip, :address) %>">
+        <%= form.label :ip, "Enter IP address (IPv4 only)", class: "govuk-label" %>
+        <%= form.text_field :address, id: :address, autocomplete: "off", class: "govuk-input" %>
+      </div>
+      <%= form.hidden_field :location_id, value: @location.id %>
```

Comment:
This is the contentious line.

Reply:
I like it, again, it's simple and clear.

Maybe I'm missing something though. Can you see problems with doing it this way?

Reply:
Hidden fields are generally a bad idea as you can manipulate the html to post a different ID. And then on top of this I'm using one controller action for these two forms, but their responsibility should probably be split between two different controllers.

But I think I'll stick with this solution for now, it gets the user flow right, then refactor to two controllers afterwards.